### ISSUE TITLE
VoiceOver does not read the correct character

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#88832]
Catalina, have VoiceOVer, Verbosity > Text > When Moving the Cursor : Speak Text to the Right of Cursor

1. Navigate over a sentence character by character using arrows in VS Code
2. When you navigate left the character that is the left of the cursor is read out, not the character that is right 🐛  

Navigating right works fine

Verbosity > Text > When Moving the Cursor :Speak text that the cursor passes
Works fine, however users prefer that the next character gets read out.

It is an upstream issue in Chrome: https://bugs.chromium.org/p/chromium/issues/detail?id=1043061

fyi @ivanfetch

**Comment by ivanfetch** [comment]
@isidorn Yes - this is the same behavior I see in VS Code with Mojave - when navigating left, the character to the left of the cursor is read instead of the character under the cursor.

Note when I previously stated that this behavior did not exist in Chrome, I had tested the Chrome address bar, which is a different type of edit field than that presented on a web page. I **can** reproduce this in Chrome on google.com in the search edit field.

**Comment by isidorn** [comment]
@ivanfetch great that we have figured out it's the same issue.
Since it is a Chrome issue this unfrotunetly means that we can not fix it right away but we have to wait for Chrome to fix it. Best would be to follow what is going on here https://bugs.chromium.org/p/chromium/issues/detail?id=1043061 and I will try to contact somebody on the Chrome side so they know that issue is important to us.

**Comment by isidorn** [comment]
This is fixed in the latest VS Code. Thus closing and assigning to this milestone so somebody besides me also verifies.

**Comment by isidorn** [comment]
Ivan confirmed via email that this is indeed fixed -> verified
